### ISSUE TITLE
Frontend gutenberg styles?

### ISSUE THREAD

**Issue by thetwopct** [joshuaiz/plate#15]
Sorry for the potentially dumb question but what does this frontend gutenberg.css file do? What should be in it?
Only realised it was even on the frontend as GT Metrix threw back a problem saying the file is so small it should be inline...

from  `functions.php`
```
plate_gutenberg_styles() {
    wp_enqueue_style( 'plate-gutenberg-styles', get_theme_file_uri( '/library/css/gutenberg.css' ), false, '1.0', 'all' );
}
```

**Comment by joshuaiz** [comment]
No worries @thetwopct. That file would be where you would put specific front-end styles for any Gutenberg blocks that your theme or plugins might add. 

It's currently empty by default so unless you've added something to _gutenberg.scss (which is also currently empty) you can remove the file. 

If my memory serves me correctly, I previously had some styles there in an earlier version of Plate so CodeKit (my preprocessor) auto generates css files from SCSS files so that's why it is there.

If you *do* remove the file, make sure to comment out the enqueue in `functions.php` so you don't get an error.

And you are also welcome to add Gutenberg styles to any of the breakpoint stylesheets if you need but some people might want to remove those (like you) if they are not using Gutenberg at all hence we split Gutenberg-related styles into their own file.

**Comment by thetwopct** [comment]
Ah OK. I tend to design front end and then sort Gutenberg editor styles for client so I don't think I'll use it much. Thanks for quick reply 🙏  

I updated my custom plate from your code a few weeks back so just discovering a few little bits. 

I was in a hurry so didn't document at the time, but I remember I found an issue with the _gutenberg.scss file being ignored by CodeKit (because of the _) - I had to put it in /scss/ and rename without the _ for it to be processed. I think one of the _editor files was the same. Just FYI 👍
